components/prism-elm.js

```diff
@@ -0,0 +1,35 @@
+Prism.languages.elm = {
+	comment: {
+		pattern: /(^|[^-!#$%*+=?&@|~.:<>^\\\/])(--[^-!#$%*+=?&@|~.:<>^\\\/].*|{-[\s\S]*?-})/m,
+		lookbehind: true
+	},
+	char: /'([^\\']|\\([abfnrtv\\"'&]|\^[A-Z@[\]\^_]|NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|SP|DEL|\d+|o[0-7]+|x[0-9a-fA-F]+))'/,
```

Comment:
This looks quite overkill to me. Remember that Prism is not a linter.
Is there any case where a single-quoted string of characters should not be highlighted as `char`? If there isn't, you should be able to drastically simplify this regexp.

Reply:
Also, you might want to add the `greedy` flag, here.

Reply:
Yeah, upon closer inspection, I think `/'([^\\']|\\([abfnrtv']|\d+|x[0-9a-fA-F]+))'/` will do - any single character (except `'` itself) or an escape sequence which comes in a few forms. Thanks!